wire-ios-request-strategy/Sources/Request Strategies/Client Message/ClientMessageRequestStrategy.swift

```diff
@@ -90,22 +90,31 @@ extension ClientMessageRequestStrategy: InsertedObjectSyncTranscoder {
     typealias Object = ZMClientMessage
 
     func insert(object: ZMClientMessage, completion: @escaping () -> Void) {
-        WireLogger.messaging.debug("inserting message", attributes: object.logInformation)
+        let messageLogAttributes = MessageLogAttributes(context: context)
+
+        Task {
+            let logInformation = await messageLogAttributes.logAttributes(object)
+            WireLogger.messaging.debug("inserting message", attributes: logInformation)
+        }
```

Comment:
Before some logs like here access `.logInformation` without `context.perform`. Even though it works for those cases, copying it to other places might crash the app.
Now it always ensures to access properties via `context.perform`.

Reply:
Done this way, thanks